data/categories.json

```diff
@@ -12,7 +12,7 @@
     "title": "Rust Language Server (RLS)",
     "description": "The engine behind the Rust IDE experience. Manages builds and data about Rust programs to provide features like 'goto def' and 'type on hover'.\n\nNote that the impl period testing issues are about implementing testing frameworks, not just writing tests.",
     "repository": "rust-lang-nursery/rls",
-    "labels": [],
+    "labels": ["good first issue"],
```

Comment:
I think this change should be in tab-category.json, rather than here, since we only want it to apply to starter issues on the RLS, not all issues.

Reply:
Done! My bad. Pushed as #42